api/src/main/java/com/my/api/IRetrofitApi.kt

```diff
@@ -1,9 +1,9 @@
-package com.my.core.data
+package com.my.api
 
-import com.my.core.domain.JourneyPlannerResult
+import com.my.core.domain.JourneyPlannerResultDomainModel
 import retrofit2.Call
 
-interface IJourneyPlannerRepository {
+interface IRetrofitApi {
```

Comment:
this cannot be called IRetrofitApi. It has to be specific to the journey planner